CMakeLists.txt

```diff
@@ -1,11 +1,9 @@
-cmake_minimum_required(VERSION 2.8.7)
+cmake_minimum_required(VERSION 3.0.2)
```

Comment:
We cannot accept this change, since Travis CI is based on Ubuntu 12.04 with CMake **2.8.7**.